README.md

```diff
@@ -126,3 +127,66 @@ Example : put libvlc to `native/x86`/`native/x64`
     <VlcLib86TargetDir>native/x86</VlcLib86TargetDir>
   </PropertyGroup>
 ```
+
+## Exclude some plugins from copy
+Sometimes, you want to build a minimal package for one of these reasons:
+- Reduce your package size
+- Remove attacking surface of your software by limiting its features
+- Speed up build times : Fastest copies are those that do not occur.
+- Just because you don't need that plugin, it's cleaner not to copy it.
+
+In your csproj, you can exclude some of the plugins by including them in a
+`VlcExcludeWindowsPlugins` item group.
+That exclusion will apply to all windows builds of libvlc (x86 and x64).
+
+There is not such inclusion/exclusion mechanism for Android because libvlc is built as one monolithic library on this platform.
+
+A few things to note:
+
+- It's in an `ItemGroup`, not in a `PropertyGroup` as before
```

Comment:
What is "it"? What is "before"? I think this could be clearer.

Reply:
Removed that statement, wasn't really useful as long as you pay attention to what is written (or what you copy/paste)